### ISSUE TITLE
PHP Fatal error:  Class 'OCP\Backgroundjob' not found -> Upgrade 9.0.4 to 9.1

### ISSUE THREAD

**Issue by JB1985** [owncloud/music#516]
I get an error on upgrade from 9.0.4 to 9.1

Command: php occ upgrade

Update 3rd-party app: music
PHP Fatal error:  Class 'OCP\Backgroundjob' not found in /var/customers/webs/web12/owncloud/apps/music/appinfo/app.php on line 38

**Comment by MU7L3Y** [comment]
Me too.
Also trying to enable the app it tells me "this app cannot be enabled because it makes the server unstable."

**Comment by andral** [comment]
Same here, trying to add the music addon on a fresh 9.1 installation.

```
{
  "reqId": "V5XSSzMuUE9wE7HQy86SkAAAAAE",
  "remoteAddr": "192.168.10.1",
  "app": "PHP",
  "message": "Class 'OCP\\Backgroundjob' not found at /var/www/html/owncloud/apps/music/appinfo/app.php#38",
  "level": 3,
  "time": "2016-07-25T08:48:11+00:00",
  "method": "GET",
  "url": "/index.php/apps/files",
  "user": "luca"
}
```
Running on centos 7
```
rpm -qa | grep owncl
owncloud-files-9.1.0-1.1.noarch
owncloud-deps-php5-9.1.0-1.1.noarch

```

**Comment by gnarula** [comment]
This is due to the PSR-4 changes in core introduced during v9.1. It's been fixed in `master` and should be resolved in the next release (due once #517 is merged)

**Comment by gnarula** [comment]
Should be fixed in [0.3.12](https://github.com/owncloud/music/releases/tag/v0.3.12). Closing this for now. Feel free to reopen this in case it still persists.
